dev.robiot.xclicker.yml

```diff
@@ -0,0 +1,33 @@
+app-id: dev.robiot.xclicker
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: xclicker
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --socket=ssh-auth
+  # - --device=all
+  - --filesystem=~/.config
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
Same about these two?

Reply:
Ditto here.

Reply:
ping @robiot

Reply:
still doesn't explain really why you need each of these d-bus names.

Reply:
WIll remove